realtime/streams.py

```diff
@@ -23,6 +23,10 @@ def put_nowait(self, item):
 class AudioStream(Stream):
     type = "audio"
 
+    def __init__(self, sample_rate: int = 8000):
```

Comment:
how does this 8k sample rate compare with 16k on websocket?

Reply:
added comment to remove in future version